### ISSUE TITLE
Creating cluster on GKE does not work anymore

### ISSUE THREAD

**Issue by sebgoa** [bitnami/cabin#25]
_From @sebgoa on July 18, 2017 13:21_

We can add clusters that already exists.

but the create button is not there yet...

iOS 10.3.1

_Copied from original issue: skippbox/cabin-sw#129_

**Comment by sebgoa** [comment]
_From @kemcake on July 18, 2017 16:29_

Maybe an issue with the permissions, do you have right access where you want to create the cluster?

**Comment by sebgoa** [comment]
yes, I am just not seeing any + button to be able to create the cluster.

I will double check on other projects.

**Comment by sebgoa** [comment]
@kemcake this is still no doable in 0.4.0 , there used to be a + icon in the GKE view where we could create a cluster. this seems to have disappeared on iOS.

**Comment by kemcake** [comment]
The + button is there, but only if you have the right permissions on the project.
See https://github.com/bitnami/cabin/blob/4864566752b3686e64317017d036dbd8c9e39ae0/src/components/Clusters/ClustersNewGoogle.js#L182 to check if the permissions corresponds to your case?
